docs/OMS-shell-bundle.md

```diff
@@ -0,0 +1,47 @@
+  3. tar file - the tar file that contains the .deb/.rpm files
+  4. Install type - the value is "DPKG" for a .deb bundle, "RPM" for an .rpm bundle, blank for an all-inclusive bundle
+  
+This script edits the bundle file i.e. [bundle_skel.sh](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/bundle/bundle_skel.sh) and enters hard-coded values for the variables: TAR_FILE, &nbsp; OMI_PKG, &nbsp; OMS_PKG,  &nbsp;DSC_PKG,  &nbsp;SCX_PKG,  &nbsp;SCRIPT_LEN,  &nbsp;SCRIPT_LEN_PLUS_ONE  &nbsp;and &nbsp; INSTALL_TYPE. After these values are in place, the script creates the bundle in the target directory.
+
+The [Makefile](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/build/Makefile) is where the bundle creation is executed. Here, the .rpm and .deb packages are built using the installbuilder. The files from these packages along with the oss-kits are copied over to create the tar files.The create_bundle.sh script is then run to create the omsagent shell bundle in the target directory using the tar file. All bundles are put in the target directory once the build is complete.
+
+### Installing the bundle:
+  **All bundle operations:**
+```
+Options:
+  --extract              Extract contents and exit.
+  --force                Force upgrade (override version checks).
+  --install              Install the package from the system.
+  --purge                Uninstall the package and remove all related data.
+  --remove               Uninstall the package from the system.
+  --restart-deps         Reconfigure and restart dependent service(s).
+  --source-references    Show source code reference hashes.
+  --upgrade              Upgrade the package in the system.
+  --version              Version of this shell bundle.
+  --version-check        Check versions already installed to see if upgradable.
+  --debug                use shell debug mode.
+  --collectd             Enable collectd.
+
+  -w id, --id id         Use workspace ID <id> for automatic onboarding.
+  -s key, --shared key   Use <key> as the shared key for automatic onboarding.
+  -d dmn, --domain dmn   Use <dmn> as the OMS domain for onboarding. Optional.
+                         default: opinsights.azure.com
+                         ex: opinsights.azure.us (for FairFax)
+  -p conf, --proxy conf  Use <conf> as the proxy configuration.
+                         ex: -p [protocol://][user:password@]proxyhost[:port]
+
+  -? | -h | --help       shows this usage text.
+
+```
+  **To install and onboard :**
+```
+sudo sh ./omsagent-1.2.0-39.universal.x86.sh --install –w <workspaceid> -s <shared key>
+```
+When running this command, before actual installation, the script will check if the correct bundle is being installed based on the processor architecture (x64 vs. x86). It will also check if the kit type being referenced to by the bundle is correct (rpm vs. deb). If either of these tests fail, the script will cleanup and exit without installation.
```

Comment:
Prefer that you line-wrap this at about 70 bytes rather than having one very long line. This makes it easier to see minor differences down the road.

You can do that in emacs with `<esc>q`.